specs/0100-private-projects.md

```diff
@@ -39,9 +42,10 @@ All instances of the "latest activity" feature, throughout the website, should a
 
 | Role | Impact |
 | -- | -- |
-| Locale Manager | Can see all private projects of locales they are managing |
-| Translator | Can see all private projects of locales they are translating |
-| Contributor | Cannot access private projects unless they are given explicit permission |
+| Project Manager | Can see all projects |
+| Locale Manager | Can see all projects visible to "Selected Users" for locales they are managing |
+| Translator | Can see all projects visible to "Selected Users" for locales they are translating |
```

Comment:
I'd think that we would want the opposite to be true, if we used this feature at Mozilla.
We'd use it to restrict projects to contributors under NDA, for example, and most locale managers aren't. Let alone translators.

Reply:
This changes everything again, then. :-) Is there a use case for this?

We can go back to the simple Private/Public switch if this also works for Wanadev. If it doesn't, we'll need a checkbox to include locale managers and translators or not.

@BThibaut Is it OK for you if Locale Managers and Translators are *not* automatically granted permission to access the private projects and you need to manually select them?

Reply:
Yes it is ok for us, this will be a little extra step to set up a project but not a regular action.

Reply:
Thanks! We've now made it easier - there will be a check to include Locale Managers and Translators.